usaspending_api/settings.py

```diff
@@ -19,6 +19,10 @@
 # User-specified timeout limit for streaming downloads
 DOWNLOAD_TIMEOUT_MIN_LIMIT = 10
 
+API_MAX_DATE = '2020-09-30'
```

Comment:
Yeah, I thought it was far enough into the future for now. It could be 2099-09-30 and it wouldn't change anything. It is for the date range combine logic